### DISCUSSION TITLE
Simple web realisation

### DISCUSSION THREAD

**Discussion by MrDrone-ru** [CoderLine/alphaTab#987] [Q&A] [answered]
Hi!
Can anyone share a simple js+html realisation of AT? Just clientside, like in tutorial. Interested in those functions that are not in the guide (and present on the AT main page): speed & volume control, open files.
Of course you can say: rtfm noob.
And of course I will go to read if there is not a person who has already "invented the wheel".

**Comment by Danielku15** [comment, marked as answer]
Hi. 
alphaTab is meant to be an SDK to be integrated in applications. Considering that I doubt that there is a fully generic implementation by someone who allows enough flexibility to really serve the needs in your project. The tutorial is supposed to give you enough inputs to kickstart your own development of integrating alphaTab into your project as you want to build it to present music notation to your users.  

There is the idea to maybe one day really provide an all-in-one component with a pre-built UI that might give a certain flexibility of customization but priorities are currently rather on shipping new platforms and features.

**Reply by MrDrone-ru** [reply]
Ok, i got it. 
I hope you don't mind if I leave a link to my project here later (as an answer to my own question). I think it will be extremely useful for those who will deal with AT api + web dev from scratch.

**Comment by MrDrone-ru** [comment]
So, that's it! [http://dronx.byethost16.com/attest.html](http://dronx.byethost16.com/attest.html)
Based on tutorial, now it can apply & change volume, mute & solo tracks (which are reading from tab file) and change playback speed and rewind & Space also work properly. Also it shows used midi instruments using jzz-midi-gm script.
Hope, somebody find it useful)) if so leave a link to my tab project mealntabs.ru
Thx for attention.
